juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -135,17 +135,17 @@ healthbot {
                 description "Sets health based on Fan RPM percentage";			
                 frequency 1offset;
                 /*
-                 * Sets color to red when RPM is more than 80%.
+                 * Sets color to green when RPM is less than 50%.
                  * 
                  */				
-                term above-high-threshold {
+                term normal {
                     when {
-                        greater-than "$rpm-percent" "$high-threshold";
-                    }
+                        less-than-or-equal-to "$rpm-percent" "$low-threshold";
+                    }				
                     then {
                         status {
-                            color red;
-                            message "$fan-name RPM percent $rpm-percent($measurement RPM) is greater than $rpm-high-threshold";
+                            color green;
+                            message "$fan-name RPM percent is $rpm-percent";
```

Comment:
As it is normal. Please add normal in message e.g. $fan-name RPM percent ($rpm-percent) is normal

Reply:
Done